app/addons/documents/routes-documents.js

```diff
@@ -317,9 +318,9 @@ function(app, FauxtonAPI, Documents, Changes, Index, DocEditor, Databases, Resou
 
     viewFn: function (databaseName, ddoc, viewName) {
       var params = this.createParams(),
-          urlParams = params.urlParams,
-          docParams = params.docParams,
-          decodeDdoc = decodeURIComponent(ddoc);
+      urlParams = params.urlParams,
+      docParams = params.docParams,
+      decodeDdoc = decodeURIComponent(ddoc);
 
```

Comment:
the intendation is funky from beginning of the diff until here. please also separate whitespace commits from feature-commits.